index.html

```diff
@@ -44,32 +44,98 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
         </a>
       </label>
       <pre class='f5 sans-serif fw5 ma0 pv2 dib overflow-x-auto w-100' id="hr-cid"></pre>
-      <div class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
+      <div class='f5 sans-serif ma0 charcoal-muted ttu f6 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
       <div class='dn' id="cid" ></div>
+      <span>
+        <p class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>
+        </br> string: base (base)(cid-version - multicodec - multihash)
+        </br> binary: cid_version - multicodec - multihash
```

Comment:
I think i know what you wanted to convey here, but description does not look intuitive. Perhaps human sentence is better here?

```suggestion
        </br>CID is encoded as String using Multibase.
        </br>After decoding Multibase, the remaining value is bytes. 
```